lib/src/patterns.dart

```diff
@@ -50,6 +50,9 @@ final listPattern =
 final tablePattern = RegExp(
     r'^[ ]{0,3}\|?([ \t]*:?\-+:?[ \t]*\|)+([ \t]|[ \t]*:?\-+:?[ \t]*)?$');
 
+/// Just keep the same with github.
```

Comment:
This comment does not describe what this pattern is. We don't have to reference GitHub.